src/client/provider.ts

```diff
@@ -23,12 +22,10 @@ export class Provider {
 
   async configure(configuration: {
     security?: SecurityValues[];
-    serviceId?: string;
```

Comment:
Removed because we don't use it or?🤷

Reply:
We don't use it. The draft spec lets maps specify which service to use, so this would be redundant.